examples/tools/android_native_run/android_symbols/AndroidSymbols.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
```

Comment:
I thought it was gone but it's not: this file is not needed and you can remove it.

Reply:
Done